src/SymfonyCache/HttpCacheAwareKernelInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\SymfonyCache;
+
+use Symfony\Component\HttpKernel\HttpCache\HttpCache;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+interface HttpCacheAwareKernelInterface extends HttpKernelInterface
+{
+    /**
+     * @param HttpCache $httpCache
+     */
+    public function setHttpCache(HttpCache $httpCache);
```

Comment:
imho this should not be part of the interface. its an implementation detail for your bootstrap. (its fine to have it in the trait though - but when bootstrapping, the user knows what trait the actual kernel class implements)

Reply:
What should not be part of this interface?

Reply:
the setter

Reply:
Ah, I see! Removed in 6964a8b3aa914bbad5442bb9c8a979505c7afe35.